parent-poms/pom.xml

```diff
@@ -816,7 +840,7 @@
                                         <exclude>org.codehaus.mojo:versions-maven-plugin:2.2</exclude>
                                     </excludes>
                                 </bannedDependencies>
-                                <dependencyConvergence/>
```

Comment:
It was commented out because it was failing for something that shouldn't be all the way up in ETL Framework  Java. I will reproduce the message.